containerized_ml_models/api/requirements.txt

```diff
@@ -0,0 +1,17 @@
+click==8.1.3
+Flask==2.2.2
+importlib-metadata==5.0.0
+itsdangerous==2.1.2
+Jinja2==3.1.2
+joblib==1.2.0
+MarkupSafe==2.1.1
+numpy==1.23.4
+pip==22.3.1
+scikit-learn==1.1.3
+scipy==1.9.3
+setuptools==65.5.1
+threadpoolctl==3.1.0
+Werkzeug==2.2.2
+wheel==0.38.4
+xgboost==1.5.1
+zipp==3.10.0
```

Comment:
Nice but this file is kinda long and I suggest only adding flask, xgboost and numpy here to make it more readable 📖